### ISSUE TITLE
color scheme issues when plotting riparian

### ISSUE THREAD

**Issue by b524198065** [jtlovell/GENESPACE#11]
Hi,

Thanks for developing the tool. I tried the code listed in the readme page and it turns out that the coloring parameter "refChrCols" did not work at all. I used refChrCols = c("#BC4F43", "#F67243"), whereas the ouptut plot only has one color of #BC4F43. Could you help indicate what's wrong? Here are my codes:

ripSouceDat <- plot_riparianHits(
  gpar, 
  refGenome = "chimp",
  invertTheseChrs = data.frame(genome = "rhesus", chr = 2),
  genomeIDs = c("chimp", "human", "rhesus"),
  labelTheseGenomes = c("chimp", "rhesus"),
  gapProp = .001,
  refChrCols = c("#BC4F43", "#F67243"),
  blackBg = FALSE, 
  returnSourceData = T, 
  verbose = F)

![image](https://user-images.githubusercontent.com/39608290/162615597-ca1b2e74-fc51-42d4-b879-421bde0820e2.png)

**Comment by jtlovell** [comment]
Hi there, 
I am unable to re-create this issue. Can you email me with the full script you used, the `sessionInfo()` and the genespace working directory you used (compressed, .zip or whatever). jlovell@hudsonalpha.org
John

**Comment by jtlovell** [comment]
Any update here? happy to help, otherwise I'll close this out.

**Comment by HongboDoll** [comment]
Hi,

Indeed I tried the "plot_riparian" function and the color works quite well. I do not know why but it seems that this plotting function is deprecated according to your codes. Are there any differences between plot_riparian and plot_riparianHits?

Thanks

**Comment by jtlovell** [comment]
tl;dr: In many cases, the functions should behave the same. For example, with just two genomes, the functions are identical. 

Details: Across multiple genomes, `plot_riparian` uses the orthogroup IDs to link genes, while `plot_riparianHits` uses pairwise syntenic anchor hits to link genes to the reference chromosomes. This is the same in many cases, but in others, with highly diverged genomes, poor assemblies, or complex orthogroup structures, the mapping across all genomes can differ from pairwise differences. This is similar to the difference between orthogroups and orthologs.
